composer.json

```diff
@@ -30,13 +30,14 @@
         "doctrine/data-fixtures": "^1.2",
         "doctrine/doctrine-bundle": "^1.6",
         "doctrine/orm": "^2.5",
-        "nelmio/alice": "^2.2",
+        "nelmio/alice": "^3.1",
         "phpspec/php-diff": "^1.1",
         "phpunit/phpunit": "^6.0",
         "polishsymfonycommunity/symfony-mocker-container": "^1.0",
         "symfony/browser-kit": "^3.2",
         "symfony/finder": "^3.2",
-        "symfony/framework-bundle": "^3.2"
+        "symfony/framework-bundle": "^3.2",
+        "theofidry/alice-data-fixtures": "^1.0@RC"
```

Comment:
When will it be published?

Reply:
I don't know, the latest RC release is from 17 days ago.

Reply:
Three days ago Theo released a stable version. Are you willing to update this PR, @angelov?